snmp/datadog_checks/snmp/data/default_profiles/apc-netbotz.yaml

```diff
@@ -0,0 +1,355 @@
+extends:
+  - apc.yaml
+sysobjectid:
+  - 1.3.6.1.4.1.5528.100.20.*
```

Comment:
kentik defines this as `1.3.6.1.4.1.5528.100.20.10.*`, why did you go back a level ?

Reply:
works for me to keep `1.3.6.1.4.1.5528.100.20.*`